CONTRIBUTING.md

```
@@ -1 +1,135 @@
+   bun run --filter '*' build 
+   ```
+
+#### n8n-kit
+
+You'll find code related to `@vahor/n8n-kit` in the [`packages/n8n-kit`](./packages/n8n-kit) directory.
+
+#### n8n-cli
+
+You'll find code related to `@vahor/n8n-kit-cli` in the [`packages/n8n-cli`](./packages/n8n-cli) directory.
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)